src/parser/csql_grammar.y

```diff
@@ -5428,6 +5558,16 @@ serial_name
 		{ $$ = $1; }
 	;
 
+synonym_name_without_dot
+	: user_specified_name_without_dot
+		{ $$ = $1; }
```

Comment:
Please add below
{  DBG_TRACE_GRAMMAR(synonym_name_without_dot, : user_specified_name_without_dot );
   $$ = $1;
}

Reply:
Thank you. Added.